src/components/Datepicker.tsx

```diff
@@ -348,12 +341,18 @@ const Datepicker: React.FC<Props> = ({
         inputRef
     ]);
 
+    const defaultContainerClassName = "relative w-full text-gray-700";
+
+    const containerClassNameOverload =
+        typeof containerClassName === "function"
+            ? containerClassName(defaultContainerClassName)
+            : typeof containerClassName === "string"
+            ? `${defaultContainerClassName} ${containerClassName}`
```

Comment:
I think there will always be problems with this line.

If in `containerClassName` we have `text-red-400` then it will conflict with `text-gray-700` of `defaultContainerClassname`.

Reply:
Yes, and that's why I offered to use [tailwind-merge](https://github.com/dcastil/tailwind-merge) to solve this issue in https://github.com/onesine/react-tailwindcss-datepicker/pull/68. To avoid adding it to the library, containerClassName (and other "class names") became functions to allow users to use twMerge (or other library) outside the datepicker.

But yes - if containerClassName is a string there still will be issue with this ... I left this to avoid errors against previous versions.

Reply:
In case containerClassName is a 

- string we want to style the container with only its own utility classes then we don't need twMerge
- function we want to add utility classes to the existing ones, then we may need twMerge to avoid conflicts.

When containerClassName is not defined, we use the default style.